UnityProject/Assets/Scripts/Tilemaps/Tiles/ObjectTile.cs

```diff
@@ -29,46 +29,46 @@ public class ObjectTile : BasicTile
 
 #if UNITY_EDITOR
 
-		private void OnValidate()
-		{
-			if (Object != null)
-			{
-				if (objectCurrent == null)
-				{
-					// if sprite already exists (e.g. at startup), then load it, otherwise create a new one
-					EditorApplication.delayCall += () =>
-					{
-						if (IsWire && wireSprite != null)
-						{
-							PreviewSprite = wireSprite;
-						}
-						else
-						{
-							//PreviewSprite = PreviewSpriteBuilder.LoadSprite(Object);
-							//PreviewSpriteBuilder.Create(Object);
-						}
-					};
-				}
-				else if (Object != objectCurrent)
-				{
-					// from one object -> other (overwrite current sprite)
-					EditorApplication.delayCall += () => { PreviewSprite = PreviewSpriteBuilder.Create(Object); };
-				}
-			}
-			else if (objectCurrent != null)
-			{
-				// setting to None object (delete current sprite)
-				GameObject obj = objectCurrent;
-				EditorApplication.delayCall += () => { PreviewSpriteBuilder.DeleteSprite(obj); };
-			}
-
-			objectCurrent = Object;
-
-			if (objectCurrent != null && objectCurrent.GetComponentInChildren<RegisterItem>() != null)
-			{
-				IsItem = true;
-			}
-		}
+		// private void OnValidate()
```

Comment:
if this really isn't needed, just remove it. Don't comment it.

Reply:
it is weirdobject tile system we don't use any more

Reply:
more reason to nuke the entire script then!